fraqbot/tests/test_shakespeare.py

```diff
@@ -22,16 +22,24 @@ def test_listening_for():
     assert LEGO.listening_for({'text': 'shake'}) is False
     assert LEGO.listening_for({'text': '!shake moin'}) is True
     assert LEGO.listening_for({'text': '!shake'}) is True
+    assert LEGO.listening_for({'text': '!Shake'}) is False
+
+
+def test_get_quote():
+    default = 'Not so much brain as ear wax.'
+    assert LEGO._get_quote('') == default
+    assert LEGO._get_quote('bad search') == default
+    assert LEGO._get_quote('scurvy') == 'I scorn you, scurvy companion.'
 
 
 @patch('Legobot.Lego.Lego.reply')
 @patch('shakespeare.Shakespeare._get_quote')
 def test_handle_check(mock_get_quote, mock_reply):
     msg = {'text': '!shake scurvy', 'metadata': {'source_user': 'bob'}}
+    LEGO.handle(msg)
 
-    # How do I check the actual results of the handle?
-    result = LEGO.handle(msg)
-    print(result)
+    mock_get_quote.assert_called_once()
```

Comment:
assert that the mocked methods (_get_quote and reply) were each called once. This verifies the logic of our handle method.